man/subscription-manager.8

```diff
@@ -647,7 +647,9 @@ The
 .B refresh
 command pulls the latest subscription data from the server. Normally, the system polls the subscription management service at a set interval (4 hours by default) to check for any changes in the available subscriptions. The
 .B refresh
-command checks with the subscription management service right then, outside the normal interval.
+command checks with the subscription management service right then, outside the normal interval. Use of the
+.B refresh
+command will clear caches related to the content access mode of the system and allow the system to retrieve fresh data as necessary.
```

Comment:
considering the addition of this sentence, wouldn't it be better to simply say that it clears all the caches? this way there is no need to write all of them, and add new ones here in case a new cache is added in the future

Reply:
We can, it's not exactly true presently. Do you think we should update the command to clear all cache's as well as the man page update? Or just the man page update?

Reply:
Ah, OK. My understanding was that `refresh` already cleared (almost) all the caches, and the addition of this PR was one that was missing.

In this case, maybe it would be worth to mention which caches are cleared, in case it is something that should be documented -- only as a minor suggestion, though.